src/components/Common/PortfolioItem.js

```diff
@@ -0,0 +1,22 @@
+import React, {Component} from 'react';
+class PortfolioItem extends Component{
+    render(){
+        return(
+            <div class="col-md-4 col-sm-6 portfolio-item">
+                    <a href="#portfolioModal1" class="portfolio-link" data-toggle="modal">
```

Comment:
Too much indentation here. Let's keep consistency.

Reply:
Also is this `link` acting a modal toggle? If so, it shouldn't be a `link`.

Reply:
Indentation is done, but the template is given by the course mentor, and I guess chapter after chapter he will advice that bad practices

Reply:
Then how can it be declared?

Reply:
It could be a `button` instead. That depends on what the element represents